state/executor.go

```diff
@@ -519,7 +523,7 @@ func (t *Transition) applyCreate(c *runtime.Contract, host runtime.Host) *runtim
 	if err := t.transfer(c.Caller, c.Address, c.Value); err != nil {
 		return &runtime.ExecutionResult{
 			GasLeft: gasLimit,
-			Err:     runtime.ErrNotEnoughFunds,
+			Err:     runtime.ErrInsufficientBalance,
```

Comment:
Why not just this?
As far as I see `transfer` method will return either this one `runtime.ErrInsufficientBalance` or some other error, so why not return correct error?
```suggestion
			Err:     err,
```

Reply:
Now it just returns errr